docs/develop/node/validator/running-a-node-macos-linux.md

```diff
@@ -181,10 +168,6 @@ Using local binary at path/to/nearcore/target/release
 Our genesis version is up to date
 Stake for user 'stakingpool.youraccount.betanet' with 'ed25519:6ftve9gm5dKL7xnFNbKDNxZXkiYL2cheTQtcEmmLLaW'
 Starting NEAR client...
-Node is running! 
+Node is running!
 To check logs call: `nearup logs` or `nearup logs --follow`
 ```
-
->Got a question?
-<a href="https://stackoverflow.com/questions/tagged/nearprotocol">
-  <h8>Ask it on StackOverflow!</h8></a>
```

Comment:
Is there a better way to get in touch? I believe StackOverflow is the best wiki platform

Reply:
> Is there a better way to get in touch? I believe StackOverflow is the best wiki platform

Is the case that most questions for validators are posted on StackOverflow?  Do the core team actively monitor Stackoverflow? 
 
cc @bowenwang1996 @frol @mikedotexe 

Here are some ways recommended to get in touch. Please let me know StackOverflow fits in here too, and I can add back the footer if StackOverflow is actively used:
https://docs.near.org/docs/develop/node/community/node-community-updates

Reply:
It is ideal if people get in touch through StackOverflow. We have the integration that pushes SO questions to Slack, and we keep the questions answered most of the time: https://stackoverflow.com/questions/tagged/nearprotocol

Reply:
Ooh, neato, I will add it back. Also thanks for commenting on Stackoverflow!